parfive/downloader.py

```diff
@@ -503,19 +520,16 @@ async def _get_http(self, session, *, url, filepath_partial, chunksize=None,
                 kwargs['proxy'] = os.environ['HTTPS_PROXY']
 
             async with session.get(url, timeout=timeout, **kwargs) as resp:
-                parfive.log.debug("%s request made to %s with headers=%s",
-                                  resp.request_info.method,
-                                  resp.request_info.url,
-                                  resp.request_info.headers)
-                parfive.log.debug("Response received from %s with headers=%s",
-                                  resp.request_info.url,
-                                  resp.headers)
+                parfive.log.debug(f"{resp.request_info.method} request made for download to "
+                                f"{resp.request_info.url} with headers={resp.request_info.headers}")
+                parfive.log.debug(f"Response received from {resp.request_info.url} with headers={resp.headers}")
```

Comment:
:/ so a bug, I will revert

Reply:
How is it a bug? Instead of parsing a string with all the interpolation done already to the logger if you pass the template and the parameters then it will interpolate it as and when it needs to. I guess you could argue not supporting the `{}` syntax is a feature request, but still not a bug.